### PR TITLE
Update token upload procedure due to folder structure changes

### PR DESCRIPTION
**Pull request by kolya182**
`BEFORE EMRGE`
- [x] - After this merged https://github.com/TrustWallet/tokens/issues/2216

### CODE REVIEW

File: tokens/add_new_token_image.md [hunk 1]
```diff
@@ -12,78 +12,144 @@ Token repository [https://github.com/TrustWallet/tokens](https://github.com/Trus
...
 
-## How to generate address for TRC10 (TRON based) token
-1. Grab `ID` for desired TRC10 token from the list https://tronscan.org/#/tokens/list (sort by TRC10)
-2. [Generate address based on TRC10 token ID](https://repl.it/@TrustWallet/generatetrustwalletaddressfortrc10token) for TRC10 token ID
-3. Use generated address and [upload](https://developer.trustwallet.com/add_new_token_image#how-to-add) an image
 
-### YouTube: Upload ERC20 Token Image to Trust Wallet:
+```
+.
+├── blockchains
```
Comment (wpank) [inline tokens/add_new_token_image.md]: Nice 😎

File: tokens/add_new_token_image.md [hunk 2]
```diff
@@ -12,78 +12,144 @@ Token repository [https://github.com/TrustWallet/tokens](https://github.com/Trus
...
+10. Press on `Create pull request`
+13. Once tests have completed and verified that your image follows all requirements, a maintainer will merge it. In 5-10 minutes your token will have the updated image instead of plain logo in Trust Wallet
+
+### Easy way for Git user
+1. Fork the repo to your own github account
+2. Clone fork and crate new branch:
+```json
+git clone git@github.com:YOUR_HANDLE/token-profile.git
+cd tokens
+git branch XXX
```
Comment (wpank) [inline tokens/add_new_token_image.md]:
```suggestion
git branch <branch_name>
```

File: tokens/add_new_token_image.md [hunk 3]
```diff
@@ -12,78 +12,144 @@ Token repository [https://github.com/TrustWallet/tokens](https://github.com/Trus
...
+git checkout XXX
```
Comment (wpank) [inline tokens/add_new_token_image.md]:
```suggestion
git checkout <branch_name>
```

File: tokens/add_new_token_image.md [hunk 4]
```diff
@@ -12,78 +12,144 @@ Token repository [https://github.com/TrustWallet/tokens](https://github.com/Trus
...
+```
+3.  Add asset to appropriate directory, here [folder strcture](#repository-structure) to help you
+4. Commit and push to your fork
+```json
+git add -A
+git commit -m “Add WHATEVER_YOU_ADDING token”
```
Comment (wpank) [inline tokens/add_new_token_image.md]:
```suggestion
git commit -m “Add <token_name>”
```

File: tokens/add_new_token_image.md [hunk 5]
```diff
@@ -12,78 +12,144 @@ Token repository [https://github.com/TrustWallet/tokens](https://github.com/Trus
...
+git push origin XXX
```
Comment (wpank) [inline tokens/add_new_token_image.md]:
```suggestion
git push origin <branch_name>
```

File: tokens/add_new_token_image.md [hunk 6]
```diff
@@ -12,78 +12,144 @@ Token repository [https://github.com/TrustWallet/tokens](https://github.com/Trus
...
-3. Press on `Fork` in the top right corner, wait for process to complete
-4. Navigate to `tokens` (or `coins`, if you're adding a coin icon) folder on your own fork
-5. Press on `Upload File` in the top right corner
-6. Choose file, make sure it follows requirements above
-7. Press on `Commit changes`
-8. Press on `New pull request` on your own fork page and submit it by pressing on `Create pull request`!
-9. Add short description including name and token symbol in a header field
-10. Press on `Create pull request`
-11. Once tests have completed and verified that your image follows all requirements, a maintainer will merge it. In 5-10 minutes your token will have the updated image instead of plain logo in Trust Wallet
+`blockchains` folder contains many subfolders and represents chains e.g. `ethreum`, `binance` ...
```
Comment (hewigovens) [inline tokens/add_new_token_image.md]:
```suggestion
`blockchains` folder contains many subfolders and represents chains e.g. `ethereum`, `binance` ...
```

File: tokens/add_new_token_image.md [hunk 7]
```diff
@@ -12,78 +12,144 @@ Token repository [https://github.com/TrustWallet/tokens](https://github.com/Trus
...
+  - optional: In `Add an optional extended description` write a comment about upload
+  - optional: adjust fork branch nam
+9. Click on `Propose changes`
+10. Press on `Create pull request`
+13. Once tests have completed and verified that your image follows all requirements, a maintainer will merge it. In 5-10 minutes your token will have the updated image instead of plain logo in Trust Wallet
+
+### Easy way for Git user
+1. Fork the repo to your own github account
+2. Clone fork and crate new branch:
+```json
```
Comment (hewigovens) [inline tokens/add_new_token_image.md]:
```suggestion
```bash
```

File: tokens/add_new_token_image.md [hunk 8]
```diff
@@ -12,78 +12,144 @@ Token repository [https://github.com/TrustWallet/tokens](https://github.com/Trus
...
+git clone git@github.com:YOUR_HANDLE/token-profile.git
+cd tokens
+git branch <branch_name>
+git checkout <branch_name>
+```
+3.  Add asset to appropriate directory, here [folder strcture](#repository-structure) to help you
+4. Commit and push to your fork
+```json
```
Comment (hewigovens) [inline tokens/add_new_token_image.md]:
```suggestion
```bash
```

File: tokens/add_new_token_image.md [hunk 9]
```diff
@@ -109,7 +109,7 @@ git checkout XXX
 4. Commit and push to your fork
 ```json
 git add -A
-git commit -m “Add WHATEVER_YOU_ADDING token”
```
Comment (vikmeup) [inline tokens/add_new_token_image.md]: let's remove git comments, who knows how to use it no need to have this

File: tokens/add_new_token_image.md [hunk 10]
```diff
@@ -1,8 +1,8 @@
-# Trust Wallet Token Images
+# Trust Wallet Assets Info
+## Overview
+Hello and welcome to Trust Wallet assets info contribution guide. We appriciate your effort to open-source. Token repository [https://github.com/TrustWallet/tokens](https://github.com/TrustWallet/tokens) (repo) is a source of images used by [Trust Wallet](https://trustwallet.com/) (TW) including:
 
-Token repository [https://github.com/TrustWallet/tokens](https://github.com/TrustWallet/tokens) (repo) source of images for:
-
-1. [ERC20](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md) tokens on Ethereum compatible networks supported by Trust Wallet such as:
+1. [ERC20](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md) tokens on Ethereum compatible networks supported by TW such as:
```
Comment (hewigovens) [inline tokens/add_new_token_image.md]: `TW` initials only used for once here, I prefer to always use Trust Wallet

Comment (kolya182) [reply tokens/add_new_token_image.md]: https://github.com/TrustWallet/developer/pull/32/commits/0f14fc799ccd9621c2f6c976163177c1f0bb1d24

**Pull request merged**